README.md

```diff
@@ -1,19 +1,94 @@
-![Build Status](https://codeship.com/projects/66094d00-42b3-0133-cd9b-1eb5f82d52fc/status?branch=master)![Code Climate](https://codeclimate.com/github/caseykim/coders_ink.png) ![Coverage Status](https://coveralls.io/repos/caseykim/coders_ink/badge.png)
+![CodersInk Logo](/public/banner.png)
 
-* [√]The ability to add an item to be reviewed.
+# Introduction
 
-* [√]The ability to rate the item and to optionally comment on it (i.e. write a review).
+CodersInk is a review website hosted at https://codersink.herokuapp.com/tattoos.  The application was designed and built by the tics project team during the Fall 2015 cohort at Launch Academy (http://www.launchacademy.com/).
 
-* [√]The ability to upvote or downvote a review. A user can only upvote or downvote once and can change their vote from up to down. This feature should utilize AJAX so that a complete page reload isn't necessary.
+![Build Status](https://codeship.com/projects/66094d00-42b3-0133-cd9b-1eb5f82d52fc/status?branch=master)
+![Code Climate](https://codeclimate.com/github/caseykim/coders_ink.png) ![Coverage Status](https://coveralls.io/repos/caseykim/coders_ink/badge.png)
 
-* [√]A sign up and authentication system for your users (with Devise).
+# Topics
 
-* [√]An ability to upload a profile photo (with Carrierwave).
+- [Features](#features)
+- [Development](#development)
+    - [Testing](#testing)
+    - [Test Development](#test-development)
+- [ER Diagram](#er-diagram)
+- [License](#license)
 
-* [√]The ability to search for items.
+# Features
 
-* [√]An admin role. Admins are able to delete comments or items if they are deemed to be inappropriate.
+- The ability to add an item to be reviewed.
+- The ability to rate the item and to optionally comment on it (i.e. write a review).
+- The ability to upvote or downvote a review. A user can only upvote or downvote once and can change their vote from up to down. This feature utilizes AJAX so that a complete page reload isn't necessary.
+- A sign up and authentication system for your users (developed with Devise).
+- An ability to upload a profile photo (developed with Carrierwave).
+- The ability to search for items.
+- An admin role. Admins are able to delete comments or items if they are deemed to be inappropriate.
+- An email is sent to the owner of an item when a new review is posted about it.
+- Pagination (developed with Kaminari).
+- Infinite scroll for the index page (developed with Javascript)
 
```

Comment:
Mention the average_rating method (checks from all reviews belonging to a tattoo) and also the top-6 page?